pkg/systemd/overview-cards/motdCard.jsx

```diff
@@ -49,14 +54,60 @@ export class MotdCard extends React.Component {
         });
     }
 
+    dialogErrorSet(text, detail) {
+        this.setState({ dialogError: text, dialogErrorDetail: detail });
+    }
+
     render() {
         if (!this.state.motdVisible)
             return null;
 
         return (
-            <Alert id="motd-box" isInline variant="default" className="motd-box"
-                   title={ <pre id="motd">{this.state.motdText}</pre> }
-                   actionClose={ <AlertActionCloseButton onClose={this.hideAlert} /> } />
+            <>
+                <Alert id="motd-box" isInline variant="default" className="motd-box"
+                       title={ <pre id="motd">{this.state.motdText}</pre> }
+                       actionClose={<>
+                           {superuser.allowed &&
+                           <Button variant="plain"
+                                   id="motd-box-edit"
+                                   onClick={() => this.setState({
+                                       motdEditMode: true,
+                                       motdTextEdited: this.state.motdText
+                                   })}
+                                   aria-label="Action">
```

Comment:
That is not a great label :) Can you put there something like `Edit motd` and don't forget to localize it as well.